docker-compose.yml

```diff
@@ -7,42 +7,44 @@ services:
     build:
       context: .
       dockerfile: Dockerfile-dev
-    command: script/docker-app.sh
+    command: docker/dev/run.sh
     environment:
-      DB_HOST: ${DB_HOST}
-      DB_USER: ${DB_USER}
-      DB_PASSWORD: ${DB_PASSWORD}
-      DB_NAME: ${DB_NAME}
+      DB_HOST: mysql
+      DB_NAME: portal
+      DB_USER: master
+      DB_PASSWORD: master
       SOLR_HOST: solr
       SOLR_PORT: 8983
-    networks:
-      - appnet
     ports:
-      - "3000:3000"
+      - 3000:3000
     volumes:
       - .:/rigse
-      - "bundle:/bundle"
+      - bundle:/bundle
   solr:
-    image: solr:5.3
-    networks:
-      - appnet
+    image: concordconsortium/docker-solr-portal
     ports:
-      - "8983:8983"
+      - 8983:8983
     volumes:
       - .:/rigse
       - data:/opt/solr/server/solr/mycores
     depends_on:
       - solrdata
-    command: /rigse/docker/start-solr-within-solr-container.sh
+    command: /bin/bash ./start-solr.sh
   solrdata:
-    image: solr:5.3
+    image: concordconsortium/docker-solr-portal
     volumes:
       - data:/opt/solr/server/solr/mycores
     user:
       root
-    command: chown solr:solr /opt/solr/server/solr/mycores
-networks:
-  appnet:
+    command: /bin/bash ./start-volume.sh
+  mysql:
```

Comment:
I don't think we should add mysql to docker as this won't allow developers to use their existing local mysql instance.  The way it is setup here the mysql data isn't saved on a persistent volume so it is lost when you bring Docker down.